### PR TITLE
MBS-12172 Rename isRealRun function

### PR DESCRIPTION
**Pull request by RuslanMingaliev**

### CODE REVIEW

**Comment by dsvoronin** [comment]
I remember how @sboishtyan changed it `isFakeOrchestratorRun` -> `isRealRun`

**Comment by sergioboi** [comment]
Hakuna matata

File: subprojects/test-runner/test-instrumentation-runner/src/main/kotlin/com/avito/android/runner/OrchestratorDelegate.kt [hunk 1]
```diff
@@ -5,6 +5,13 @@ import androidx.test.internal.runner.RunnerArgsAccessor
 
 interface OrchestratorDelegate {
 
+    @Deprecated(
```
Comment (sergioboi) [inline subprojects/test-runner/test-instrumentation-runner/src/main/kotlin/com/avito/android/runner/OrchestratorDelegate.kt:8]: @RuslanMingaliev Do you really need this change?

Comment (RuslanMingaliev) [reply subprojects/test-runner/test-instrumentation-runner/src/main/kotlin/com/avito/android/runner/OrchestratorDelegate.kt:8]: don't think so

**Pull request closed without merging**